src/main/js/controls/Line/Line.js

```diff
@@ -184,6 +188,47 @@ class Line extends GraphContent {
         return this;
     }
 
+    /**
+     * @inheritdoc
+     */
+    reflow(graph, graphData) {
+        this.config.values = graphData.values;
+        this.dataTarget = processDataPoints(graph.config, this.config);
+        const lineSVG = graph.svg
+            .select(`g[aria-describedby="${graphData.key}"]`)
+            .selectAll(`.${styles.line}`)
+            .data([this.dataTarget]);
+        drawDataLines(graph.scale, graph.config, lineSVG.enter());
+        lineSVG.exit().remove();
+
+        if (graph.config.showShapes) {
+            const currentPointsPath = graph.svg
+                .select(`g[aria-describedby="${graphData.key}"]`)
+                .selectAll(`.${styles.pointGroup}`)
+                .data(this.dataTarget);
+            currentPointsPath.exit().remove();
+            const pointPath = graph.svg
+                .select(`g[aria-describedby="${graphData.key}"]`)
+                .select(`.${styles.currentPointsGroup}`)
+                .selectAll(`[class*="${styles.point}"]`)
+                .data(getDataPointValues(this.dataTarget));
+            drawDataPoints(graph.scale, graph.config, pointPath.enter());
```

Comment:
So once the points are redrawn in the reflow. where the curve connecting the points get added. do we need to handle that in the current logic itself ?